src/main/resources/db/changelog/changes/v1.0.4/sql/derived-tables/create-view-purchase-order-lines.sql

```diff
@@ -0,0 +1,56 @@
+CREATE OR REPLACE VIEW drv_purchase_order_line_details
+AS
+SELECT po.id,
+    po.jsonb ->> 'poLineNumber'::text AS purchase_order_line_number,
+    po.jsonb ->> 'receiptStatus'::text AS purchase_order_line_receipt_status,
+    po.jsonb ->> 'paymentStatus'::text AS purchase_order_line_payment_status,
+    (po.jsonb -> 'metadata'::text) ->> 'createdDate'::text AS purchase_order_line_created_date,
+    (po.jsonb -> 'metadata'::text) ->> 'updatedDate'::text AS purchase_order_line_update_date,
+    (po.jsonb -> 'metadata'::text) ->> 'createdByUserId'::text AS purchase_order_line_creater_id,
+	concat((user_details2.jsonb -> 'personal'::text) ->> 'lastName'::text, ', ',
```

Comment:
Probably not a huge deal, but we could use concat_ws along with NULLIF to handle null values for user's first name, etc. I doubt we'll encounter many users without a first name, but it is technically possible to create one.

Reply:
An example of this is seen in the item_effective_call_number field of drv_item_details